### PR TITLE
show supplementary grading

### PR DESCRIPTION
**Pull request by travisreed-wf**
image

### CODE REVIEW

**Comment by travisreed-wf** [comment]
ACTION REQUIRED:
```python
alter table task_response add column cognitive_grade float;
```

File: src/taskBuilder/views.py [hunk 1]
```diff
@@ -79,6 +79,8 @@ def post(self):
         task.course = models.Course.query.filter_by(id=courseID).first()
         task.duedate = datetime.datetime.fromtimestamp(taskDueDate/1000.0)
         task.supplementary = json.dumps(data.get('supplementary'))
+        print data.get('supplementary')
+        print data.keys()
```
Comment (AJH693) [inline src/taskBuilder/views.py]: get rid of these prints

File: src/templates/tasks/tasks_js.js [hunk 2]
```diff
@@ -236,9 +236,11 @@
 
   function submitClicked(element) {
       var numSupplementaryMinTimes = parseInt(nextSupplementaryMinTimeID.split("minTime")[1]);
+      console.log(numSupplementaryMinTimes);
       for(i = 0 ; i < numSupplementaryMinTimes; i++){
-        supplementaryInformationMinTimes[i] = $("#minTime" +i).val();
+        supplementaryInformationMinTimes["supplementary" + i] = parseInt($("#minTime" +i).val());
       }
+      console.log(supplementaryInformationMinTimes);
```
Comment (AJH693) [inline src/templates/tasks/tasks_js.js]: these logs

**Comment by AJH693** [comment]
get rid of those prints and you good + 1

**Comment by travisreed-wf** [comment]
hartman reviewed

**Pull request merged**